agent.go

```diff
@@ -127,12 +127,15 @@ type agent interface {
 	// start will start the agent on the host.
 	start() error
 
+	// isStarted will check the agent is started.
+	isStarted() bool
+
 	// exec will tell the agent to run a command in an already running container.
 	exec(podID string, contID string, cmd Cmd) error
 
-	// StartPod will tell the agent to start all containers related to the Pod.
+	// startPod will tell the agent to start all containers related to the Pod.
 	startPod(config PodConfig) error
 
-	// StopPod will tell the agent to stop all containers related to the Pod.
+	// stopPod will tell the agent to stop all containers related to the Pod.
```

Comment:
Those 2 are separate changes, so I'd appreciate if you could put that into a separate patch as well.